### DISCUSSION TITLE
Autogrowing Input?

### DISCUSSION THREAD

**Discussion by KrychuTM** [tailwindlabs/tailwindcss#4507] [Help] [answered]
Is there a way I can autogrow an input as it's text field fills up in tailwindcss??

**Comment by simonswiss** [comment, marked as answer]
Hey!

I _don't think_ it's possible to do this with just CSS, but this can be achieved with a little bit of JavaScript.

Here's an interesting article on the topic:

https://css-tricks.com/auto-growing-inputs-textareas/
